algo/src/main/java/org/neo4j/graphalgo/results/metaPathComputationResults/GetSchemaResult.java

```diff
@@ -14,7 +12,7 @@
     public final String schema;
     public final String reverseLabelDictionary;
 
-    private GetSchemaResult(ArrayList<HashSet<Pair>> schema, IntIntHashMap reverseLabelDictionary) {
+    private GetSchemaResult(ArrayList<HashSet<Pair>> schema, HashMap<Integer, Integer> reverseLabelDictionary) {
         Gson gson = new Gson();
         this.schema = gson.toJson(schema);
         this.reverseLabelDictionary = gson.toJson(reverseLabelDictionary);
```

Comment:
maybe write the result to disk rather here than somewhere else?

Reply:
Then we can only use getSchema by using the Proc. Is that fine with us?